addons/resource.language.en_gb/resources/strings.po

```diff
@@ -7712,7 +7712,19 @@ msgctxt "#13476"
 msgid "16 bit per channel"
 msgstr ""
 
-#empty strings from id 13477 to 13504
+#. Some Android devices have a bug and do not correctly report the capabilities (profile/level combinations it supports) of the codec component.
+#: system/settings/settings.xml
+msgctxt "#13477"
+msgid "Bypass the capabilities reported by the codec - MediaCodec"
+msgstr ""
+
+#. Description of setting with label #13477 "Bypass the capabilities reported by the codec - MediaCodec"
+#: system/settings/settings.xml
+msgctxt "#13478"
+msgid "Some devices do not correctly report the capabilities supported by the codecs, this setting allows you to skip this check."
```

Comment:
maybe it's better to specify that side effects could occur?
```suggestion
msgid "Some devices do not correctly report the capabilities supported by the codecs, this setting allows you to skip this check, to consider that in some cases it could lead to other playback problems."
```

Reply:
It should be made clear that this is a setting for specific cases only and, as you point out, it could have undesirable effects on the video playback.